### PR TITLE
Add a phone number to contact page

### PR DESCRIPTION
**Pull request by SerbanCiprian**
## Why 

We want to provide a phone number so interested parties can reach us.

## What 
- [x] Add Larry's cell phone number to the Contact page in front of the email address

## Notes

- The contact page is available here: https://www.ascentcore.com/contact.html
- The phone number is `(703) 867-5055`
- The mockups for the contact page are available here: https://app.moqups.com/yUwnyd2GEz/edit/page/a878506ad (ask TJ to give you access if you don't have any)

Example crop from mockups:

Screenshot 2021-04-04 at 20 25 18

### CODE REVIEW

File: _includes/homepage/intro.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <div class="container">
     <div class="intro__inner">
       <div class="intro__copy">
-        <h2 class="intro__title">We take it personally</h2>
+        <h2 class="intro__title">We take it personally!</h2>
```
Comment (hcristea) [inline _includes/homepage/intro.html:6]: @SerbanCiprian, the exclamation was added as a test? Looks like it's outside the scope of the work.

Comment (SerbanCiprian) [reply _includes/homepage/intro.html:6]: Yeah, sorry, was added as a test. I forgot to change.

Comment (hcristea) [reply _includes/homepage/intro.html:6]: Ok, no problem. Please change it back in your next PR. Thanks! Enjoy you evening :)
